### PR TITLE
Switch to Inter font

### PR DESCRIPTION
**Pull request by petrosagg**
After this change we can generate Normal.java and Bold.java directly

### CODE REVIEW

File: src/main/java/de/bottlecaps/railroad/core/FontWidth.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <html>
     <head>
         <meta http-equiv="content-type" content="text/html; charset=utf-8"/>
+<style>
+ @import url('https://fonts.googleapis.com/css2?family=Encode+Sans+Expanded:wght@100;300&family=Fira+Code&family=Inter:wght@300;400;500;700&display=swap');
```
Comment (benesch) [inline src/main/java/de/bottlecaps/railroad/core/FontWidth.html:5]: Did you mean to include encode sans and fira code here?

**Review by benesch — APPROVED** [review]
I don't 100% follow this, but seems good, and very much if it works, ship it. (My question is mostly: why didn't it just work like this to start? Is there something we're not seeing? But, again, if it works, who cares.)

**Comment by petrosagg** [comment]
@benesch the generation of `Normal.java` and `Bold.java` consists of a local computation using `FontWidth.html` that generates a list of widths for the codepoints specified. Then you submit this list of numbers to https://bottlecaps.de/rex/ which generates the java file that seems to be doing some sort of compression/hashmap because its `get(int codepoint)` function does [these weird bitshifts](https://github.com/MaterializeInc/rr/blob/master/src/main/java/de/bottlecaps/railroad/core/Normal.java#L18) to index an array.

I couldn't find its source code, I only know its version has changed based on a comment it adds at the top of the file. When you generated it it was REx v5.50 and the current one is v5.53. Plus the resulting SVGs seemed wrong and I can't debug this.

So this PR gets rid of this web service that generates the weird Java class and just embeds all the widths with no compression as a list. Then `get(int codepoint)` is a [simple index to that array](https://github.com/MaterializeInc/rr/blob/inter-font/src/main/java/de/bottlecaps/railroad/core/Normal.java#L27) which worked like a charm.

File: src/main/java/de/bottlecaps/railroad/core/FontWidth.html (hunk 1 above)
Comment (petrosagg) [reply src/main/java/de/bottlecaps/railroad/core/FontWidth.html:5]: ah, I just copy pasted our CSS rule from the main repo

**Pull request merged**